apps/ledger-live-desktop/src/renderer/screens/manager/FirmwareUpdate/index.tsx

```diff
@@ -88,35 +91,33 @@ const FirmwareUpdate = (props: Props) => {
     });
 
     setFirmwareUpdateOpened(true); // Prevents manager from reacting to device changes (?)
-    setDrawer(
-      UpdateModal,
-      {
-        withAppsToReinstall:
-          !!installed &&
-          installed.length > 0 &&
-          manager.firmwareUpdateWillUninstallApps(deviceInfo, device.modelId),
-        withResetStep: manager.firmwareUpdateNeedsLegacyBlueResetInstructions(
-          deviceInfo,
-          device.modelId,
-        ),
-        onDrawerClose,
-        status: modal,
-        stepId: stepId,
-        installed: installed,
-        firmware: firmware,
-        deviceInfo: deviceInfo,
-        device: device,
-        error: error,
-        deviceModelId: deviceSpecs.id,
-        setFirmwareUpdateOpened,
-        setFirmwareUpdateCompleted,
-      },
-      {
-        preventBackdropClick: true,
-        forceDisableFocusTrap: true,
-        onRequestClose,
-      },
-    );
+    const updateModalProps: UpdateModalProps = {
+      withAppsToReinstall:
+        !!installed &&
+        installed.length > 0 &&
+        manager.firmwareUpdateWillUninstallApps(deviceInfo, device.modelId),
+      withResetStep: manager.firmwareUpdateNeedsLegacyBlueResetInstructions(
+        deviceInfo,
+        device.modelId,
+      ),
+      onDrawerClose,
+      status: modal,
+      stepId: stepId,
+      installed: installed,
+      firmware: firmware,
+      deviceInfo: deviceInfo,
+      device: device,
+      error: error,
+      deviceModelId: deviceSpecs.id,
+      setFirmwareUpdateOpened,
+      setFirmwareUpdateCompleted,
+      shouldReloadManagerOnCloseIfUpdateRefused: true,
```

Comment:
Make sure QA will test this scenario with a normal fw update from the manager to be safe